### ISSUE TITLE
Keyboard navigation for the dialogues with option buttons

### ISSUE THREAD

**Issue by goplayerjuggler** [microsoft/vscode#159293]
Type: <b>Feature Request</b>

VS code has several different types of message boxes, or dialogues, with option buttons. I don't know what their proper name is. I find it frustrating that there is no keyboard navigation for these dialogues (except for the Escape key, to dismiss them).

It would be good to be able to select an option in these dialogues via the keyboard, without needing a mouse. I suggest a new command: "Focus next dialogue" and then be able to cycle through focussing the option buttons via the Tab key.

I give a repro below for opening one of these dialogues.

1. Create a read-only test file and open it in VS Code, via the following PowerShell script
``` PowerShell
$filePath = 'c:\temp\test.txt'
New-Item -Path $filePath -ItemType File -Force
Set-ItemProperty -Path $filePath -Name IsReadOnly -Value $true
& code $filepath
``` 
2. In VS Code, enter some text in the file and attempt to save it. Get a dialogue with option buttons like this:
![image](https://user-images.githubusercontent.com/18676432/186901097-a928508d-0660-4667-b508-0b5c5ddd9f73.png)
It should be possible to select one of the options (Overwrite / Save As... / Discard) with the keyboard (Tab and Enter keys)

VS Code version: Code 1.70.2 (e4503b30fc78200f846c62cf8091b76ff5547662, 2022-08-16T05:35:13.448Z)
OS version: Windows_NT x64 10.0.19044
Modes:

**Comment by isidorn** [comment]
"F1 > show notifications". `Tab` key to move focus between buttons.
You can also assign a keyboard shortcut to this command.

@goplayerjuggler is there something that is missing in this flow? If yes, what are your suggestion for improvements? Thanks

**Comment by goplayerjuggler** [comment]
@isidorn thanks, it's fine. I missed that. So I'm closing this issue.

**Comment by isidorn** [comment]
Ok. If you have other suggestions on how to improve here do let us know. Thanks
